packages/docs/docs/.vuepress/config.js

```diff
@@ -35,6 +35,7 @@ module.exports = ctx => ({
       apiKey: '3a539aab83105f01761a137c61004d85',
       indexName: 'vuepress'
     }) : null,
+    smoothScroll: true,
```

Comment:
It's TBD that we should enable `smoothScroll` for official docs site. cc @ulivz

Reply:
I support trying out new features in our official docs.